ARCs/arc-0086.md

```diff
@@ -0,0 +1,149 @@
+
+These individuals can participate in the designation and approval of proposals submitted to the Algorand Expert Governance process.
+
+An xGov is associated with an Algorand Address subscribing to the Algorand Expert Governance by acknowledging the xGov Registry (Application ID: _TBD_).
+
+Once the xGov Registry confirms the acknowledgement, the xGov is eligible to acquire _voting power_.
+
+### xGov Voting Power
+
+The _voting power_ assigned to each xGov is equal to the number of blocks proposed by its Algorand Address over a past period of blocks.
+
+### xGov Committee
+
+An xGov Committee is a group of eligible xGovs that have acquired voting power in a block period.
+
+An xGov Committee is defined by the following parameters:
+
+- xGov Registry creation block `Bc` (`uint64`);
+- Inception period `I` (`uint64`), is the number of blocks, after `Bc`, dedicated to the initial xGov Registry acknowledgement by the xGovs;
+- Committee period start `Bi` (`uint64`), it **MUST** be `0 mod 1,000,000` and `Bi ≥ Bc + I`;
+- Committee period end `Bf` (`uint64`), it **MUST** be `0 mod 1,000,000` and `Bf > Bi`;
+- Selected list of xGovs, each element is a pair of address and vote (`(bytes[32], uint64)`);
+- Total committee members (`uint64`), is the size of the selected list; 
+- Total committee votes (`uint64`), is the sum of votes in the selected list.
+
+The xGov Committee selection is repeated periodically to select new xGov Committees over time.
+
+### xGov Committee Selection Procedure
+
+Given the xGov Committee parameters `(I, Bc, Bi, Bf)`, the selection is executed with the following procedure:
+
+1. Collect all proposed blocks in the range `[Bi; Bf]` to build the `potential_committee` (note that not all the Block Proposers are eligible as xGov). For each Block Proposer address in the `potential_committee`, assign a voting power equal to the number of blocks proposed in the range `[Bi; Bf]`.
+
+2. Collect all the _eligible_ xGovs in the range `[Bc + I; Bf]` to build the `eligible_xgovs` list by:
+    - Adding to the list all xGov with a _subscription_ acknowledgements on the xGov Registry in the range;
+    - Removing from the list all xGov with a _unsubscription_ acknowledgements on the xGov Registry in the range;
+
+3. Filter `potential_committee` ∩ `eligible_xgovs` to obtain the final `committee`.
+
+### Representation
+
+The xGov Committee selection **MUST** result in a JSON with following schema:
+
+```json
+{
+  "title": "xGov Committee",
+  "description": "Selected xGov Committee with voting power and validity",
+  "type": "object",
+  "properties": {
+    "xGovs": {
+      "description": "xGovs with voting power, sorted lexicographically with respect to addresses",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "address": {
+            "description": "xGov address used on xGov Registry",
+            "type": "string"
+          },
```

Comment:
@tasosbit should we specify an encoding for the `address`?

Reply:
@cusma, can we still pursue even with your question pending?